src/components/RadioButtonGroup/index.tsx

```diff
@@ -1,36 +1,43 @@
 import React, { useEffect, useState } from 'react';
 import { Text, View } from 'react-native';
-import { RadioButton } from 'react-native-paper';
+import RadioGroup, { RadioButtonProps } from 'react-native-radio-buttons-group';
 
 import { SETTING_WEIGHT_ACTIVE_TYPE } from '../../constants';
 import { dataWeight } from '../../storage';
 import { styles } from './styles';
 
+const radioButtonsData: RadioButtonProps[] = [
+  {
+    id: '1',
+    label: '   Metric ',
+    color: '#008000',
+    value: SETTING_WEIGHT_ACTIVE_TYPE,
+  },
+  {
+    id: '2',
+    label: ' Imperial ',
+    color: '#008000',
+    value: 'imperial',
+  },
+];
```

Comment:
Radio button group should be reusable, so pass this as a prop from the place where you call `<RadioButtonGroup radioGroup={"HERE SHOULD BE YOUR ARRAY PASSED"}/>`